### PR TITLE
Updated Web Workflow utility script env.py to use settings.toml instead of .env files

### PR DESCRIPTION
**Pull request by makermelissa**
With CP 8 beta 6 out now, this change is required. The rest of the guide is already updated.

### CODE REVIEW

File: Web_Workflow_Quickstart/env.py [hunk 1]
```diff
@@ -6,79 +6,87 @@
...
+def enumerate_toml_files():
     found_files = []
-    all_files = os.listdir("/")
+    all_files = os.listdir(SETTINGS_FOLDER)
     for current_file in all_files:
-        if current_file[:4] == ".env" and len(current_file) > 4:
-            found_files.append(current_file)
+        if (
+            current_file[:2] != "._"
+            and current_file[-5:] == ".toml"
```
Comment (dhalbert) [inline Web_Workflow_Quickstart/env.py:18]: I like `startswith` and `endswith` for this kind of checking.

```suggestion
            not current_file.startswith("._")
            and not current_file.endswith("toml")
```

File: Web_Workflow_Quickstart/env.py [hunk 2]
```diff
@@ -6,79 +6,87 @@
...
-                env.write(line)
-        env.close()
+                settings.write(line)
         print("Done. Hard resetting board...")
         microcontroller.reset()
     except RuntimeError:
         print("You can't change the env file with this script while USB is mounted")
 
 
 # Return a prettier name than the env file
```
Comment (dhalbert) [inline Web_Workflow_Quickstart/env.py:60]:
```suggestion
# Return a prettier name than the toml filename
```

**Review by dhalbert — APPROVED** [review]
Thanks! Minor suggestions; your choice to change or not.

File: Web_Workflow_Quickstart/env.py (hunk 1 above)
Comment (makermelissa) [reply Web_Workflow_Quickstart/env.py]: Cool, I didn't know about that function.

File: Web_Workflow_Quickstart/env.py [hunk 3]
```diff
@@ -6,79 +6,87 @@
...
+def enumerate_toml_files():
     found_files = []
-    all_files = os.listdir("/")
+    all_files = os.listdir(SETTINGS_FOLDER)
     for current_file in all_files:
-        if current_file[:4] == ".env" and len(current_file) > 4:
-            found_files.append(current_file)
+        if (
+            not current_file.startswith("._")
+            and not current_file.endswith(".toml")
```
Comment (dhalbert) [inline Web_Workflow_Quickstart/env.py:18]: Whoops, I introduced a bug! 
```suggestion
            and current_file.endswith(".toml")
```

**Review by dhalbert — CHANGES_REQUESTED** [review]

**Comment by dhalbert** [comment]
maybe re-test :)

**Comment by makermelissa** [comment]
Tested successfully.

**Review by dhalbert — APPROVED** [review]

**Pull request merged**